### ISSUE TITLE
Precompiled binaries for minor electron versions do not exist

### ISSUE THREAD

**Issue by AndrewMuise** [grpc/grpc-node#1201]
### Problem description
Precompiled binaries for electron versions 6.1.x and 7.1.x do not exist, resulting in the need to rebuild from source.

Expect them to exist within https://console.cloud.google.com/storage/browser/node-precompiled-binaries.grpc.io/grpc/?project=grpc-testing as they do for other versions

### Reproduction steps
Using
```
  "dependencies": {
    "grpc": "^1.23.3",
  },
  "devDependencies": {
    "electron": "6.1.5",
  }
```
In the `package.json` results in `npm` rebuilding as the precompiled binary does not exist

### Environment
Node version: 12.31.1
Package name and version: grpc@1.23.3 and up

**Comment by nicolasnoble** [comment]
These got released recently, and we haven't added then to our build matrix yet.

**Comment by murgatroid99** [comment]
Precompiled binaries for those versions are available in grpc 1.24.4.
